CIP-????/README.md

```diff
@@ -0,0 +1,43 @@
+---
+CIP: ???
+Title: Cardano Governance Security
+Category: Ledger
+Status: Proposed
+Authors:
+    - Richard McCracken <rickymac68@icloud.com>
+    - Andrew Westberg <andrewwestberg@gmail.com>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/490
+Created: 2023-03-27
+License: CC-BY-4.0
+---
+
+
+# Abstract
+
+As Cardano community led governance comes online, security of the governance mechanisme must be addressed in a manner so that all participants are aware of the security weaknesses and how each weakness can be addressed. The guidelines contained in the CIP are relevant to Constittional Committe members, Delegation Representatives, Stake Pool Operators, voters, and Ada holders at large. The guidlines also contain as to how wallet developers, dApp developers, and infrastructure developers in general to help build situational awareness so that as development progresses in these areas, consideration in design is given to governance security.
```

Comment:
```suggestion
As Cardano community led governance comes online, security of the governance mechanism must be addressed in a manner so that all participants are aware of the security weaknesses and how each weakness can be addressed. The guidelines contained in the CIP are relevant to Constittional Committe members, Delegation Representatives, Stake Pool Operators, voters, and Ada holders at large. The guidelines also contain as to how wallet developers, dApp developers, and infrastructure developers in general to help build situational awareness so that as development progresses in these areas, consideration in design is given to governance security.
```